### PR TITLE
feat: Adding init to terraform_tflint hook

### PR DESCRIPTION
**Pull request by andrew-glenn**
Put an `x` into the box if that apply:

- [ ] This PR introduces breaking change.
- [ ] This PR fixes a bug.
- [ ] This PR adds new functionality.
- [x ] This PR enhances existing functionality.

### Description of your changes
Just a quick `tflint --init` before running tflint to ensure plugins are installed.

### CODE REVIEW

File: hooks/terraform_tflint.sh [hunk 1]
```diff
@@ -36,6 +36,11 @@ function per_dir_hook_unique_part {
 
   # Print checked PATH **only** if TFLint have any messages
   # shellcheck disable=SC2091,SC2068 # Suppress error output
+  $(tflint --init 2>&1) 2> /dev/null || {
+    common:colorify "yellow" "tflint init:"
+    tflint --init
+  }
```
Comment (yermulnik) [inline hooks/terraform_tflint.sh:42]: The `init` command seems to be a one-off command if there were no changes to `tflint` plugins and hence the first run of `tflint --init` will do the thing and invalidate the second run and hide the initial `tflint` plugins initialization.
I think it's worth to do something like the below:
```
TFLINT_INIT=$(tflint --init 2>&1) 2> /dev/null || {
  local exit_code=$?
  common:colorify "yellow" "tflint init:"
  echo "$TFLINT_INIT"
  # return exit code to common::per_dir_hook
  return $exit_code
}

This needs proper testing as I wrote this from the top of my head w/o running the script to ensure the code is Okay.

Comment (andrew-glenn) [reply hooks/terraform_tflint.sh]: Done, and you were spot on.

**Comment by andrew-glenn** [comment]
@yermulnik  - feedback addressed.

**Comment by mbainter** [comment]
@andrew-glenn I think you need to capitalize the `a` in `Adding` in your PR title for it to pass that still failing check.

**Review by yermulnik — APPROVED** [review]
LGTM

**Comment by yermulnik** [comment]
@antonbabenko @MaxymVlasov I haven't tested this PR, though it looks good and should not break anything. The failed PR check fix is here #358 358

**Comment by antonbabenko** [comment]
This PR is included in version 1.65.0 :tada:

**Comment by MaxymVlasov** [comment]
Ok, I'll create separate PR

**Pull request merged**